apps/website-new/docs/zh/practice/bridge/react-bridge.mdx

```diff
@@ -209,16 +224,23 @@ export declare interface ProviderParams {
 * `bridgeInfo`
   * type:
 ```tsx
-type ProviderFnParams<T> = {
+interface ProviderFnParams<T> {
   rootComponent: React.ComponentType<T>;
   render?: (
     App: React.ReactElement,
     id?: HTMLElement | string,
   ) => RootType | Promise<RootType>;
+  createRoot?: (
+    container: Element | DocumentFragment,
+    options?: CreateRootOptions,
+  ) => Root;
 };
 ```
 
   * 作用: 用于传递根组件
+    * `rootComponent`: 需要被远程加载的根组件
+    * `render`: 自定义渲染函数，用于自定义渲染逻辑
+    * `createRoot`: 自定义 createRoot 函数，用于自定义 React 根节点的创建方式，可用于兼容不同版本的 React 或实现特定的渲染行为
```

Comment:
Wouldn't it be more advantageous to translate it into English, or keep both the code and comments in English? This way, we would reach a larger audience and also maintain a consistent language standard in the code.

Reply:
@douglaszaltron Oh, I got what you mean. I think the thing is, I forgot the English document of this part! Yes you are right,you've made a very good point. We'll definitely add English docs for this part, and I think for english users we can just turn to the english version of the website and it will all be English documents. Thank again!